schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE "patients"(
+    "id" INT GENERATED ALWAYS AS IDENTITY,
+    "name" VARCHAR(255) NOT NULL,
+    "date_of_birth" DATE NOT NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE "medical_histories"(
+    "id" INT GENERATED ALWAYS AS IDENTITY,
+    "admitted_at" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    "patient_id" INTEGER NOT NULL,
```

Comment:
Kindly you have to add INDEX for this FOREIGN KEY, as the requirements say.

Reply:
Asked change implemented